content/en/docs/setup/install/istioctl/index.md

```diff
@@ -223,40 +211,6 @@ resources must be removed manually.
 
 {{< /warning >}}
 
-## Show differences in manifests
```

Comment:
maybe move this somewhere in the advanced section rather than completely removing?